src/VisualStudio/IntegrationTests/VisualStudioIntegrationTests.csproj

```diff
@@ -0,0 +1,38 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\..\..\build\Targets\VSL.Settings.targets" />
+  <PropertyGroup>
+    <ProjectGuid>{E5A55C16-A5B9-4874-9043-A5266DC02F58}</ProjectGuid>
+    <OutputType>Library</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <RootNamespace>Roslyn.VisualStudio.Integration.UnitTests</RootNamespace>
```

Comment:
Can you rename the .csproj to match the assembly name?  I think @davkean was working on something to do this so we might as well start creating projects this way, too.  Otherwise it's too hard to navigate in the Solution Explorer.